q2_diversity/tests/test_beta.py

```diff
@@ -111,7 +111,7 @@ def test_bioenv(self):
             index_fp = os.path.join(output_dir, 'index.html')
             self.assertTrue(os.path.exists(index_fp))
             self.assertTrue('metadata1' in open(index_fp).read())
-            self.assertFalse('metadata2' in open(index_fp).read())
+            self.assertTrue('metadata2' in open(index_fp).read())
```

Comment:
Can you update these tests to be a little more specific? i.e. that `metadata2` column is in the "non-numeric" warning?

Can you also add a unit test that tests out the zero-variance functionality?